db/citations.yml

```diff
@@ -8455,6 +8455,40 @@ llan;RobustnessTestSpacegroupMining22:
   wwwemail: ''
   wwwpub: ''
   year: '2022'
+llan;sbm;prm24:
+  ackno: ''
+  author:
+    - Ling Lan
+    - Qiang Du
+    - Simon J. L. Billinge
+  doi: ''
+  entrytype: article
+  facility: ''
+  grant: DE-SC0024141
```

Comment:
doeneutron23 for grant

Reply:
This is different from the paper acknowledgment. In the paper, we used DE-SC0024141 which you have confirmed twice. Do we want to change the paper acknowledgment?

Reply:
no.  What is needed here is the regolith database ID of the grant.